docs/concepts/what-is-ipfs.md

```diff
@@ -1,89 +1,90 @@
-- **Makes it harder to censor content.** Because files on IPFS can come from many places, it's harder for anyone (whether they're states, corporations, or someone else) to block things. We hope IPFS can help provide ways to circumvent actions like these when they happen.
-- **Can speed up the web when you're far away or disconnected.** If you can retrieve a file from someone nearby instead of hundreds or thousands of miles away, you can often get it faster. This is especially valuable if your community is networked locally but doesn't have a good connection to the wider internet. (Well-funded organizations with technical expertise do this today by using multiple data centers or CDNs — [content distribution networks](https://en.wikipedia.org/wiki/Content_delivery_network). IPFS hopes to make this possible for everyone.)
+- [Centralization](#centralization)
+- [Performance](#performance)
+- [Scalability](#scalablity)
+- [Link rot](#link-rot)
+- [Security](#security)
+- [Data sovereignty and ownership](#data-sovereignty)
+- [Verifiable off-chain storage](#verifiable-off-chain-storage)
+- [Local-first software](#local-first-software)
+- [Vendor lock-in](#vendor-lock-in)
 
-That last point is actually where IPFS gets its full name: the **InterPlanetary File System**. We're striving to build a system that works across places as disconnected or as far apart as planets. While that's an idealistic goal, it keeps us working and thinking hard, and almost everything we create in pursuit of that goal is also useful here at home.
+### Centralization
 
-## Content addressing
+IPFS is an open-source, decentralized system that eliminates the need for a central authority, making it more resilient and censorship-resistant than traditional centralized systems. No single entity or person controls, manages or owns IPFS; rather, it is a community-maintained project with multiple implementations of the protocol, multiple tools and apps leveraging that protocol, and multiple users and organizations contributing to it's design and development.
 
-::: callout
-For a beginner-friendly primer on why cryptographic hashing and content addressing matter, take a look at ProtoSchool's tutorial, [Content Addressing on the Decentralized Web](https://proto.school/content-addressing).
-:::
+### Performance
 
-What about that link to the aardvark page above? It looked a little unusual:
+IPFS provides faster access to data by storing it at multiple locations, and allowing users to access it from the nearest location using content addressing instead of <VueCustomTooltip label="Data identified and linked to by it's location. An example is HTTP." underlined>location-based addressing</VueCustomTooltip>. In other words, because data can be addressed based on it's contents, a node on the network can fetch that data from _any_ other node in the netork that has the data; thus, performance issues like latency are reduced. 
```

Comment:
**IPFS itself doesn't store in multiple locations.**

The way I think about it is that "IPFS enshrines the freedom to replicate data on the network" and that "IPFS enables data to be retrieved from multiple locations"